website/docs/ephemeral-resources/organization_token.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: Ephemeral: tfe_organization_token"
+description: |-
+  Generates a new organization token that is guaranteed not to be written to
+  state.
+---
+
+# Ephemeral: tfe_organization_token
+
+Terraform ephemeral resource for managing a TFE organization token. This
+resource is used to generate a new organization token that is guaranteed not to
+be written to state. Since organization tokens are singleton resources, using this ephemeral resource will replace any existing organization token.
```

Comment:
```suggestion
be written to state. Since organization tokens are singleton resources, using this ephemeral resource will replace any existing organization token, including those managed by `tfe_organization_token`.
```

Reply:
Addressed in 93f913be.